stream-compositions/events/product-events/src/main/resources/events/egress/arrangement-completed-event.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "productGroups": {
+      "description": "Arrangement internal id",
```

Comment:
need to change this to productGroups ID

Reply:
This is for a single arrangement , i wont get the productGroupId